BitwardenShared/UI/Platform/Application/Extensions/UITabBarControllerExtensionsTests.swift

```diff
@@ -0,0 +1,97 @@
+import SnapshotTesting
+import XCTest
+
+@testable import BitwardenShared
+
+// MARK: - UITabBarControllerTests
+
+class UITabBarControllerTests: BitwardenTestCase {
+    var module: MockAppModule!
+    var coordinator: TabCoordinator!
+    var rootNavigator: MockRootNavigator!
+    var vaultDelegate: MockVaultCoordinatorDelegate!
+    var settingsDelegate: MockSettingsCoordinatorDelegate!
+    var errorReporter: MockErrorReporter!
+    var vaultRepository: MockVaultRepository!
+    var subject: UITabBarController!
+
+    // MARK: Setup & Teardown
+
+    override func setUp() {
+        super.setUp()
```

Comment:
⛏️ Could you add a `tearDown()` method that `nil`s out the local properties?

Reply:
ah good catch!  i told myself to add it before finishing out the file but clearly that was missed!  thanks